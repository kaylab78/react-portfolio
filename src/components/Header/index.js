import React from "react";

function Header(props) {
  return (
    <div>
    <h2>
      <a href="/">Kayla Backus</a>
    </h2>
    {props.children}
    </div>
  );
}

export default Header;
