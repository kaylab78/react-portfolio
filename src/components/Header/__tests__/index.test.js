import React from 'react';
import { render, cleanup, screen } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Header from '..';

afterEach(cleanup);

describe('Header component', () => {
    it('renders', () => {
        render(<Header />);
    });

    it('matches snapshot', () => {
        const { asFragment } = render(<Header />);
        expect(asFragment()).toMatchSnapshot();
    });
});