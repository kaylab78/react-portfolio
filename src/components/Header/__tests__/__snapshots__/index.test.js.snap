// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component matches snapshot 1`] = `
<DocumentFragment>
  <div>
    <h2>
      <a
        href="/"
      >
        Kayla Backus
      </a>
    </h2>
  </div>
</DocumentFragment>
`;
